src/wizard/wizard-dialog.ts

```diff
@@ -51,6 +52,7 @@ https://docs.google.com/drawings/d/1LAYImcreQdcUxtBt10K76FFypMGRTwu1tGBEERfAWkE/
 const BOARD_GENERIC_ESP32 = "esp32dev";
 const BOARD_GENERIC_ESP8266 = "esp01_1m";
 const BOARD_GENERIC_ESP32S2 = "esp32-s2-saola-1";
+const BOARD_GENERIC_ESP32S3 = "esp32-s3-devkitc-1";
```

Comment:
I chose this board as it is one of two listed on Espressif docs:
https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/hw-reference/esp32s3/user-guide-devkitc-1.html

But it is the only board listed in ESPhome boards list:
https://github.com/esphome/esphome/blob/dbbbba3cf8ff5ad73f200c085f0a7f84c6be9aa1/esphome/components/esp32/boards.py#L1135